zeebe/protocol-impl/src/test/java/io/camunda/zeebe/protocol/impl/JsonSerializableToJsonTest.java

```diff
@@ -2529,6 +2531,55 @@ final var record = new ClockRecord().offsetBy(30);
         """
       },
       /////////////////////////////////////////////////////////////////////////////////////////////
+      //////////////////////////////////// AuthorizationRecord ////////////////////////////////////
+      /////////////////////////////////////////////////////////////////////////////////////////////
+      {
+        "Authorization record",
+        (Supplier<AuthorizationRecord>)
+            () ->
+                new AuthorizationRecord()
+                    .setAuthorizationKey(1L)
+                    .setOwnerKey("owner")
+                    .setOwnerType(AuthorizationOwnerType.USER)
+                    .setResourceKey("resource")
+                    .setResourceType("type")
+                    .setPermissions(List.of("permission")),
+        """
+        {
+          "authorizationKey": 1,
+          "ownerKey": "owner",
+          "ownerType": "USER",
+          "resourceKey": "resource",
+          "resourceType": "type",
+          "permissions": ["permission"]
+        }
+        """
+      },
+      /////////////////////////////////////////////////////////////////////////////////////////////
+      ///////////////////////////////// Empty AuthorizationRecord /////////////////////////////////
+      /////////////////////////////////////////////////////////////////////////////////////////////
+      {
+        "Empty AuthorizationRecord",
+        (Supplier<AuthorizationRecord>)
+            () ->
+                new AuthorizationRecord()
+                    .setOwnerKey("owner")
+                    .setOwnerType(AuthorizationOwnerType.USER)
+                    .setResourceKey("resource")
+                    .setResourceType("type")
+                    .setPermissions(List.of("permission")),
```

Comment:
🔧 Permissions is not required in this record, so it's not truely empty atm. Let's remove this and verify permissions get serialized to an empty list.